server/controllers/finance/reports/budget_analytical/index.js

```diff
@@ -43,9 +47,98 @@ async function report(req, res, next) {
     const reportColumn = [];
     const reportFootColumIncome = [];
     const reportFootColumExpense = [];
+    let configurationReferences = [];
 
     const fiscalYear = await Fiscal.lookupFiscalYear(fiscalYearId);
 
+    if (includeSummarySection) {
+      const cashboxesIds = _.values(params.cashboxesIds);
```

Comment:
Isn't this already an array?  What does `_.values()` add?

Reply:
The Lodash function is important for this section this method is used to retrieve only the values, in the form of an array and the query uses the IN clause, here is the error we get when we do not use Lodash
![image](https://github.com/user-attachments/assets/7a5261a7-4856-4721-a350-57e19d96f257)